docs/get-started/getting-started-workload-qualification.md

```diff
@@ -0,0 +1,143 @@
+the potential GPU plan and if there are any not-supported features on a CPU cluster.
+Basically it can return output which is the same as the driver logs with `spark.rapids.sql.explain=all`.
+
+1. In `spark-shell`, add the `rapids-4-spark` and `cudf` jars into --jars option or put them in Spark related classpath.
+
+   For example:
+
+   ```bash
+   spark-shell --jars /PathTo/cudf-<version>.jar,/PathTo/rapids-4-spark_<version>.jar
+   ```
+
+2. Test if the class can be successfully loaded or not.
+
+   ```scala
+   import com.nvidia.spark.rapids.ExplainPlan.explainPotentialGPUPlan
+   ```
+
+3. Enable optional RAPIDS Accelerator related parameters based on your setup.
+
+   Enabling optional parameters may allow more operations to run on the GPU but please understand the meaning and risk 
+   of above parameters before enabling it. Please refer to [configs doc](../configs.md) for details of RAPIDS 
+   Accelerator parameters.
+   
+   For example, if your jobs have `double`/`float`/`decimal` operators together with some Scala UDFs, you can set 
+   the following parameters:
+   
+   ```scala
+   spark.conf.set("spark.rapids.sql.incompatibleOps.enabled", true)
+   spark.conf.set("spark.rapids.sql.variableFloatAgg.enabled", true)
+   spark.conf.set("spark.rapids.sql.decimalType.enabled", true)
+   spark.conf.set("spark.rapids.sql.castFloatToDecimal.enabled",true)
+   spark.conf.set("spark.rapids.sql.castDecimalToFloat.enabled",true)
+   spark.conf.set("spark.rapids.sql.udfCompiler.enabled",true)
+   ```
+
+4. Run the function `explainPotentialGPUPlan` on the query DataFrame.
+
+   For example:
+
+   ```scala
+   scala> val df_multi=spark.sql("SELECT value82*value63 FROM df2 union SELECT value82+value63 FROM df2")
+   df_multi: org.apache.spark.sql.DataFrame = [(CAST(value82 AS DECIMAL(9,3)) * CAST(value63 AS DECIMAL(9,3))): decimal(15,5)]
+
+   scala> val output=com.nvidia.spark.rapids.ExplainPlan.explainPotentialGPUPlan(df_multi)
+   scala> println(output)
+   ```
+
+   Below are sample driver log messages starting with `!` which indicate the unsupported features in this version:
+   
+   ```
+   !Exec <ProjectExec> cannot run on GPU because not all expressions can be replaced
+     !Expression <Multiply> (promote_precision(cast(value82#30 as decimal(9,3))) * promote_precision(cast(value63#31 as decimal(9,3)))) cannot run on GPU because The actual output precision of the multiply ilarge to fit on the GPU DecimalType(19,6)
+   ```
+
+This log can show you which operators(on what data type) can not run on GPU and what is the reason.
+If it shows a specific RAPIDS Accelerator parameter which can be turned on to enable that feature, you should firstly 
+understand the risk and applicability of that parameter based on [configs doc](../configs.md) and then enable that 
+parameter and try the tool again.
+
+Since its output is directly based on specific version of `rapids-4-spark` jar, the gap analysis is pretty accurate.
```

Comment:
Formatting and rewording
```suggestion
This log can show you which operators (on what data type) can not run on GPU and the reason.
If it shows a specific RAPIDS Accelerator parameter which can be turned on to enable that feature,
you should firstly understand the risk and applicability of that parameter based on [configs
doc](../configs.md) and then enable that parameter and try the tool again.

Since its output is directly based on specific version of `rapids-4-spark` jar, the gap analysis is
pretty accurate.
```